test/262/262test.js

```diff
@@ -78,7 +77,7 @@ assert.notSameValue = function (actual, unexpected, message) {
     message += ' ';
   }
 
-  message += 'Expected SameValue(«' + String(actual) + '», «' + String(unexpected) + '») to be false';
+  message += 'Expected SameValue(<' + String(actual) + '>, <' + String(unexpected) + '>) to be false';
```

Comment:
The benefit of keeping the unicode angle brackets is that that is what test262 prints itself (like when run through [test262-harness](https://github.com/bterlson/test262-harness))